### PR TITLE
Create fullstack_developer_at_TOP-IX.html

### PR DESCRIPTION
**Pull request by pendolare**
Just created a new job position

### CODE REVIEW

File: jobs/fullstack_developer_at_TOP-IX.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Full-Stack Developer
+company: TOP-IX Consortium
+url: http://www.top-ix.org
+location: Turin, Italy
+contract: permanent
+contact: staff@top-ix.org
+created: !!timpestamp '2018–02-01’
```
Comment (lordmauve) [inline jobs/fullstack_developer_at_TOP-IX.html]: Typo in `timestamp` here

Comment (lordmauve) [reply jobs/fullstack_developer_at_TOP-IX.html]: And also the closing quote is a smartquote - I think that's the error that Travis CI is complaining about.

**Comment by hartwork** [comment]
This merge request is 2 months old now and has pending requests for changes. I was about to proposing closing with author timeout but the listing is still active at https://www.top-ix.org/en/2017/11/03/job-description-full-stack-developer-top-ix/.
How about fixing the tiny issues ourselves, putting that longer source link in, adjusting the creation timestamp back to 2017-11-03 and then be done and move on?

**Comment by salimfadhley** [comment]
I'm closing this. Get me on Hangouts - salimfadhley@gmail.com.

I'm happy to take you through the process of setting up a job ad.

**Pull request closed without merging**